scripts/package/truenas/tn.config

```diff
@@ -222,6 +222,7 @@ CONFIG_BPF_SYSCALL=y
 CONFIG_ARCH_WANT_DEFAULT_BPF_JIT=y
 # CONFIG_BPF_JIT_ALWAYS_ON is not set
 CONFIG_BPF_JIT_DEFAULT_ON=y
+CONFIG_BPF_UNPRIV_DEFAULT_OFF=y
```

Comment:
I don't recall this being a required flag in order to use eBPF. We should still be able to run eBPF instrumentation with this new setting ?

Reply:
Yes. Either as root, or manually enabling it via sysctl. This only changes sysctl default toward disabling it for unprivileged users.

Reply:
This option is also set by default in new Debian kernel config.